scripts/build-webpack.js

```diff
@@ -33,7 +36,7 @@ const compiler = webpack({
     }),
   ],
   output: {
-    filename: `yarn-${version}.js`,
+    filename: `[name]-${version}.js`,
```

Comment:
Mmm. But (as per your comment in the main thread) it may be simpler to get something out the door that's published manually, in which case independent versioning might be easier.